test/lender/Cast.sol

```diff
@@ -7,7 +7,7 @@ library Cast {
     /// @dev Safely cast an uint256 to an uint128
     /// @param n the u256 to cast to u128
     function u128(uint256 n) internal pure returns (uint128) {
-        require (n <= type(uint128).max, ''); // TODO err msgs
+        require (n <= type(uint128).max, 'failed to cast');
```

Comment:
let's take the messages back out -- i want to formally decide the wording for all error messages. we can prob just look at the swivel contracts and word it like those.